packages/components/src/dropdown-menu/types.ts

```diff
@@ -59,6 +59,7 @@ type ToggleProps = Partial<
 	>
 > & {
 	as?: React.ElementType | keyof JSX.IntrinsicElements;
+	'data-toolbar-item': boolean;
```

Comment:
I think this should be optional?

```suggestion
	'data-toolbar-item'?: boolean;
```

Reply:
Done.